app/src/main/java/org/standardnotes/notes/MainActivity.kt

```diff
@@ -25,6 +26,8 @@ class MainActivity : AppCompatActivity() {
         fab.setOnClickListener { view ->
             (supportFragmentManager.findFragmentById(R.id.noteListFrag) as NoteListFragment).startNewNote()
         }
+
+        SyncManager.startSyncTimer()
```

Comment:
I think starting and stopping the sync should happen on resumed. I think this code will have the sync loops running from the MainActivity and the NoteActivity at the same time, which is unlikley to be what we want.